INSTALL.md

```diff
@@ -1,50 +1,26 @@
 ---
 ## Mysterium VPN node (Any OS with Docker)
 
-Most convenient way requiring least configuration is to run a node using docker. 
+Most convenient way requiring least configuration is to run a node using [Docker](https://docs.docker.com/install/). 
 All node versions are available through docker hub: 
  
 https://hub.docker.com/r/mysteriumnetwork/mysterium-node/
-### Installation
-
-Read on how to setup docker repository for Debian [here](https://docs.docker.com/install/linux/docker-ce/debian/) 
-and for Ubuntu [here](https://docs.docker.com/install/linux/docker-ee/ubuntu/).
 
-##### Debian / Ubuntu
+### Fetching and running docker image
 ```bash
-sudo apt-get install docker-ce
+sudo docker run --cap-add NET_ADMIN --publish "1194:1194" --name mysterium-node -d mysteriumnetwork/mysterium-node --agreed-terms-and-conditions
 ```
 
-##### CentOS / RedHat
-
-Read on how to setup docker repository for CentOS [here](https://docs.docker.com/install/linux/docker-ce/centos/)
-and for RedHat [here](https://docs.docker.com/install/linux/docker-ee/rhel/).
-
-```bash
-sudo yum install docker-ce
-```
-
-##### Fetching and running docker image
-```bash
-sudo docker run --cap-add NET_ADMIN --net host --publish "1194:1194" --name mysterium-node -d mysteriumnetwork/mysterium-node:{VERSION} --agreed-terms-and-conditions
-```
-You can skip `{VERSION}` to fetch latest image.
-
 Note: to run server, you will have to accept terms & conditions by adding '--agreed-terms-and-conditions' command line option.
```

Comment:
Suggest to improve instruction like here: https://hub.docker.com/r/mysteriumnetwork/mysterium-node/

Reply:
Both updated.